config/environment.py

```diff
@@ -100,7 +100,8 @@
     'invalid': {'id': 0, 'order': 4, 'state': 'closed'},
     'non-compat': {'id': 0, 'order': 5, 'state': 'closed'},
     'wontfix': {'id': 0, 'order': 6, 'state': 'closed'},
-    'worksforme': {'id': 0, 'order': 7, 'state': 'closed'}}
+    'worksforme': {'id': 0, 'order': 7, 'state': 'closed'},
+    'moved': {'id': 0, 'order': 1, 'state': 'closed'}}
```

Comment:
see comment https://github.com/webcompat/webcompat.com/pull/3645#commitcomment-60534190

Reply:
Good to know, I'll change that :)